packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -453,193 +380,140 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
     lineHeight,
     letterSpacing,
     radius: {
-      xs: '4px',
-      sm: '8px',
-      md: '12px',
-      lg: '16px',
-      xl: '20px',
+      xs: '2px',
+      sm: '6px',
+      md: '8px',
+      lg: '12px',
+      xl: '16px',
     },
+    
     shadow: {
       xs: `${getCssVar(
```

Comment:
@siriwatknp I might need some help with the shadows. I've tried several ways to keep it ok both in light and dark mode, but it seems impossible to make a dark drop shadow work consistently without varying the opacity.